Offseason2022/src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -99,6 +99,7 @@ public class Climber extends SubsystemBase
 
   private Timer                           m_safetyTimer         = new Timer( ); // Safety timer for use in climber
   private double                          m_safetyTimeout;                // Seconds that the timer ran before stopping
+  private Timer                           timer                 = new Timer( );

```

Comment:
Since this is a class member variable, we should use the m_ naming convention for it.